### ISSUE TITLE
Lutris game banner not showing

### ISSUE THREAD

**Issue by Blood-Pirouette** [mmatyas/pegasus-frontend#1073]
### The issue

When lutris is added as a source for pegasus, only the lutris game icon asset appears. The game banner does not appear. This behaviour is consistent across different themes and lutris games. Steam games show all assets normally. Possibly related to: [Issue #778 ](https://github.com/mmatyas/pegasus-frontend/issues/778)

### Step-by-step to reproduce the problem.

1. Install pegasus-frontend (installed from AUR)
2. Install games on lutris (games are assigned banners and other assets automatically)
3. Open pegasus-frontend
4. Lutris game banners not showing 

### System info

OS: Arch Linux x86_64
CPU: Intel i5-10300H
GPU: Intel CometLake-H GT2
GPU: NVIDIA GeForce RTX 2060
RAM: 16GB

### Pegasus version

Latest pegasus build: alpha16-65-g6dd137e4 2023-8-25

### Pegasus log

Attached
[lastrun.log](https://github.com/mmatyas/pegasus-frontend/files/12713195/lastrun.log)

**Comment by Blood-Pirouette** [comment]
I found that the banners and coverarts for lutris are saved under `~/.cahce/lutris/`. I copied the folders to `~/.local/share/lutris` as it is the `datadir` but the images still did not show. The file formats were .jpg initially, when I converted them to .png then they showed up.... so I think the issue has to do with how `Lutrisprovider.cpp` backend finds lutris assets and how pegasus-fe handles .jpg files (?)

**Comment by mmatyas** [comment]
Sorry for the late reply! I've checked Lutris, and apparently the path of the assets was changed in Lutris 0.5.10. However, older versions are still in use (5.9 is the one available from APT on *buntu, and 5.8 on older Debians), so ideally both paths should be supported. This will need some changes in the detection logic.

**Comment by Blood-Pirouette** [comment]
Not at all. Thank you for looking into it :) , I was not aware of that change. 5.9 and previous version also create a .cache folder so I figured just checking for the existence of that folder would not be enough. I updated the PR to check for the existence of ~/.cache/lutris/banners directory specifically and return the datadir if this was not found. Couldn't come up with a better solution lol  🤪. Anyways, thanks again for your help 👍
